Source/WebCore/platform/graphics/cocoa/CMUtilities.mm

```diff
@@ -166,45 +165,52 @@ static CFStringRef convertToCMYCbCRMatrix(PlatformVideoMatrixCoefficients coeffi
 
     auto& videoInfo = downcast<const VideoInfo>(info);
 
-    auto data = videoInfo.atomData->createCFData();
-    ASSERT(videoInfo.codecName == 'vp09' || videoInfo.codecName == 'vp08');
-    CFTypeRef configurationKeys[] = { CFSTR("vpcC") };
-    CFTypeRef configurationValues[] = { data.get() };
-    auto configurationDict = adoptCF(CFDictionaryCreate(kCFAllocatorDefault, configurationKeys, configurationValues, std::size(configurationKeys), &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks));
-
-    Vector<CFTypeRef> extensionsKeys { PAL::kCMFormatDescriptionExtension_SampleDescriptionExtensionAtoms };
-    Vector<CFTypeRef> extensionsValues = { configurationDict.get() };
-
-    if (videoInfo.colorSpace.fullRange && *videoInfo.colorSpace.fullRange) {
-        extensionsKeys.append(PAL::kCMFormatDescriptionExtension_FullRangeVideo);
-        extensionsValues.append(kCFBooleanTrue);
+    RetainPtr extensions = adoptCF(CFDictionaryCreateMutable(kCFAllocatorDefault, 5, &kCFTypeDictionaryKeyCallBacks, &kCFTypeDictionaryValueCallBacks));
+
+    if (videoInfo.atomData) {
+        RetainPtr data = videoInfo.atomData->createCFData();
+        ASSERT(videoInfo.codecName == kCMVideoCodecType_VP9 || videoInfo.codecName == 'vp08' || videoInfo.codecName == kCMVideoCodecType_H264 || videoInfo.codecName == kCMVideoCodecType_HEVC || videoInfo.codecName == kCMVideoCodecType_AV1);
+        CFStringRef keyName = [](auto codec) {
```

Comment:
```suggestion
        CFStringRef keyName = [] (auto codec) {
```

Reply:
that's against coding style https://webkit.org/code-style-guidelines/ 
right
```
[](int x) { return x; }
[this] { return m_member; }
[=]<typename T> { return T(); }
[&]<typename X>(X parameter) { return parameter; }
```

wrong:
```
[] (int x) { return x; }
[this]{ return m_member; }
[=] <typename T> { return T(); }
[&]<typename X> (X parameter) { return parameter; }
```